src/cssLanguageTypes.ts

```diff
@@ -284,6 +284,7 @@ export interface FileStat {
 export interface FileSystemProvider {
 	stat(uri: DocumentUri): Promise<FileStat>;
 	readDirectory?(uri: DocumentUri): Promise<[string, FileType][]>;
+	getContent?(uri: DocumentUri, encoding?: BufferEncoding): Promise<string>;
```

Comment:
Matches the [`RequestService`](https://github.com/microsoft/vscode/blob/6077d59e422b88df1ec5ce7b539bcf50dd7077f6/extensions/css-language-features/server/src/cssServer.ts#L65) on the language server side